docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -110,6 +110,12 @@ recovered by copying data from the snapshot repository, just like primaries of
 {search-snap} shards. In contrast, replicas of regular indices are restored by
 copying data from the primary.
 
+Searchable snapshots recover from repository snapshots. The repository and
+the associated snapshots must be available and reliable (see
+<<searchable-snapshots-reliability>> for more details), irrespective of the
+mount options. It is not allowed to remove a repository if there are any
+mounted searchable snapshots from it.
```

Comment:
I think we could be even more precise, here a suggestion (but please note that I'm very bad at writing doc so feel free to adjust):

```suggestion
Searchable snapshots use repository snapshots as their source of data. As such the repository and
the associated snapshots must be available and reliable (see <<searchable-snapshots-reliability>> for more details), irrespective of the mount options. Lost or corrupt snapshots in the repository will cause searchable snapshots shards to fail (which could stay unnoticed for some time). Elasticsearch does not allow to unregister a repository if there are any mounted searchable snapshots from it.
```

Reply:
Thanks, will incorporate!